feeds/pypi/main.go

```diff
@@ -66,24 +85,43 @@ type PubSubMessage struct {
 // Poll receives a message from Cloud Pub/Sub. Ideally, this will be from a
 // Cloud Scheduler trigger every `delta`.
 func Poll(ctx context.Context, m PubSubMessage) error {
```

Comment:
@dlorenc one more note on this one. It looks like to move to Cloud Run, we need to expose a standard http handler. In my fork, I was able to make this just the standard `Poll(w http.ResponseWriter, r *http.Request)`, then triggering it directly from cloud scheduler. If we can make that small adjustment, then we'll be in good shape to merge.